apps/web/src/data/subgraph/requests/daoMembersList.ts

```diff
@@ -0,0 +1,44 @@
+import * as Sentry from '@sentry/nextjs'
+
+import { SDK } from 'src/data/subgraph/client'
+import { CHAIN_ID } from 'src/typings'
+
+import { DaoTokenOwner_OrderBy, OrderDirection } from '../sdk.generated'
+
+interface DaoMember {
+  id: string
+  daoTokenCount: number
+  timeJoined: number
+}
+
+export const membersListRequest = async (
+  chainId: CHAIN_ID,
+  collectionAddress: string,
+  page?: number,
+  limit = 10
+): Promise<DaoMember[] | undefined> => {
+  try {
+    const data = await SDK.connect(chainId).daoMembersList({
+      where: {
+        dao: collectionAddress,
+      },
+      first: limit,
+      skip: page ? (page - 1) * limit : 0,
+      orderBy: DaoTokenOwner_OrderBy.DaoTokenCount,
+      orderDirection: OrderDirection.Desc,
+    })
+
+    if (!data.daotokenOwners) return undefined
+    return data.daotokenOwners.map((member) => ({
```

Comment:
can we change `DaoMember.id` to something more descriptive like `DaoMember.address`.
you can also pull the address from the `owner` field of DaoTokenOwner instead of `member.id.split(':')[1]`

Reply:
Yeah, that makes a lot more sense. Wonder how I missed `owner`